### PR TITLE
Feat coze

### PR DESCRIPTION
**Pull request by chyroc**
## Description

raycast extension for coze.com

## Screencast

bot_chat
bot_list
chat_history
workspace_list

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by chyroc** [comment]
@pernielsentikaer can you review this

File: extensions/coze/src/services/config.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { getPreferenceValues } from "@raycast/api";
+
+interface Preferences {
+  api_base: string;
+  debug: boolean;
+}
+
```
Comment (pernielsentikaer) [inline extensions/coze/src/services/config.tsx:7]: You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`x

File: extensions/coze/package.json [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  "license": "MIT",
+  "preferences": [
+    {
+      "name": "api_base",
+      "title": "API Base",
+      "description": "The base URL of the Coze API",
+      "type": "textfield",
+      "required": true,
+      "default": "https://api.coze.com",
+      "placeholder": "https://api.coze.com"
```
Comment (pernielsentikaer) [inline extensions/coze/package.json:20]: Will this ever change? Is a preference necessary?

Comment (chyroc) [reply extensions/coze/package.json:20]: it can set to https://api.coze.cn

Comment (chyroc) [reply extensions/coze/package.json:20]: or inner api host

File: extensions/coze/src/services/config.tsx (hunk 1 above)
Comment (chyroc) [reply extensions/coze/src/services/config.tsx]: i remove type, but it error:

```
Error: src/services/config.tsx(4,3): error TS7008: Member 'api_base' implicitly has an 'any' type.
Error: src/services/config.tsx(5,3): error TS7008: Member 'debug' implicitly has an 'any' type.
```

Comment (pernielsentikaer) [reply extensions/coze/src/services/config.tsx]: Sounds like you need to restart he TS server

image

And then you can do it like this (also note that 'await' doesn't do anything)

image

Comment (chyroc) [reply extensions/coze/src/services/config.tsx]: thanks, i fix it.

File: extensions/coze/package.json (hunk 2 above)
Comment (pernielsentikaer) [reply extensions/coze/package.json:20]: Makes sense

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**